content/docs/user-guide/pipelines/defining-pipelines.md

```diff
@@ -0,0 +1,256 @@
+# Defining Pipelines
+
+Pipelines represent data workflows that you want to **reproduce** reliably -- so
+the results are consistent. The regular pipelining process involves:
```

Comment:
usually ... or optionally (not all steps are required)

Reply:
Changing regular -> typical.